src/linearmixedmodel.jl

```diff
@@ -880,7 +919,7 @@ function StatsBase.vcov(m::LinearMixedModel{T}; corr=false) where {T}
         vv = covmat[iperm, iperm]
     end
 
-    corr ?  StatsBase.cov2cor!(vv,stderror(m)) : vv
+    corr ?  StatsBase.cov2cor!(vv, stderror(m)) : vv
```

Comment:
Did you want to redefine `stderror` to redirect to `stderror!` plus a pre-allocated scratch like `fixef` and `fixef!`?

Reply:
I suppose now that I have written `stderror!` we should use it.  I expect we will see reductions in the hundreds of nanoseconds in the execution times. :smiley: